packages/anthropic/src/anthropic-tools.ts

```diff
@@ -1,26 +1,13 @@
+import { ToolResultContent } from '@ai-sdk/provider-utils';
 import { z } from 'zod';
 
-// Copied from ai package
 type ExecuteFunction<PARAMETERS, RESULT> =
```

Comment:
is ExecuteFunction not copied in some fashion?

Reply:
couldn't find a reference, tried searching multiple ways